### DISCUSSION TITLE
showing plugins that are shared on repos like conda-forge

### DISCUSSION THREAD

**Discussion by neuromusic** [chanzuckerberg/napari-hub#101] [Ideas]
from convo w/ @tlambert03 ....

some plugins (esp those that depend on C binaries) are easier to build and distribute in conda repos like conda-forge. @goanpeca is doing some work to better support this in napari

- are there repositories besides pypi (like conda-forge) that we should index?
- might a "conda-napari" channel make sense one day?
- different repositories will have different installation methods... (e.g. `pip install` vs `conda install`) we would need the "install" dialog box to be aware of package-specific installation details. our aspirational "one click install" may be able to help ensure smooth installation, regardless of the package source

**Comment by tlambert03** [comment]
+1 would be great to simply be able to indicate that a package is on conda, even if the in-app install is still lacking.

> does a "conda-napari" channel make sense one day?

hard to imagine this ... I'd actually go so far as to say we should _only_ support packages on `conda-forge` for a while.
it would:
1. simplifies our search scope
2. simplify install routines (no need for funky channel setups)
3. add a layer of authenticity, since it requires at least a modicum of effort to get a package on conda-forge.
